src/BlackSlope.Api/Operations/Movies/MoviesController.cs

```diff
@@ -136,6 +137,7 @@ public async Task<ActionResult<MovieViewModel>> Post([FromBody] CreateMovieViewM
         [Route("api/v1/movies/{id}")]
         public async Task<ActionResult<MovieViewModel>> Put(int? id, [FromBody] MovieViewModel viewModel)
         {
+            Contract.Requires(viewModel != null);
```

Comment:
This should be handled in the validator ( _updateMovieRequestValidator) and return a custom message.

Reply:
This was added just to get rid of Analyzer warning of null check. Even if somehow viewModel is null, the validate call does check and result will say 'Movie' must not be empty then  HandleValidationFailure throws a BadRequest exception.
Question becomes is using Contracts worth it? Or should just turn off that warning?